packages/react/jest.config.js

```diff
@@ -1,11 +1,21 @@
 module.exports = {
-  preset: 'ts-jest',
-  testEnvironment: 'jsdom',
-  setupFilesAfterEnv: ['./jest.setup.ts'],
-  modulePathIgnorePatterns: ['<rootDir>/dist/'],
-  globals: {
-    'ts-jest': {
-      tsconfig: 'tsconfig.json',
+  collectCoverage: true,
+  collectCoverageFrom: ['<rootDir>/src/**/*.(ts|tsx)'],
+  // ignore coverage for top level "export", PrimitiveCatalog, and style files
+  coveragePathIgnorePatterns: [
+    '<rootDir>/src/(index|internal|legacy|PrimitiveCatalog|styles).(ts|tsx)',
+  ],
+  coverageThreshold: {
+    global: {
+      branches: 71,
+      functions: 60,
+      lines: 79,
+      statements: 79,
```

Comment:
I'm guessing this is just where our coverage is currently at?

Reply:
That's correct 😄

Reply:
Our current coverage is surprisingly higher than I thought 😂